### PR TITLE
Web console screenshot update

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Some quick comments:
- Updates all the `.png` files related to the web console.
- Notably, the "About" page in the console no longer has the cli download instructions. This information is now on the "Command Line Tools" page.  It is possible content may need to be updated to accompany the images.
- dev_guide/images/console_enter_source_uri.png ignored, could not find a reference to it in any of the text documents (can remove?)
- dev_guide/images/console_select_image.png ignored (can remove?)
- dev_guide/images/console_select_template.png ignored (can remove?)
- release_note/images/about_page.png ignored (can remove?) 

**Issues to address**:
- [ ] `architecture/infranfrastructure_components/web_console.adoc` the console nav has changed considerably, so only 2 of the bullet text remains.  I need some updated content for this piece.
- [ ] `dev_guide/new_app.adoc`the create_from_image.png has a section for request limits.  I did not add a callout number to the image.

@adellape @luciddreamz think you guys both are good to ping for review on this.  Def want your input on the todos re: content updates to match the new images

@jwforres to keep you in the loop!

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
Also, a couple project-overview updates.  I used some images that had some reference to pipelines. Do we need non-pipeline overview images? Both?

**Comment by jwforres** [comment]
@benjaminapetersen i'd say generally don't include pipeline stuff in the images unless its a page about pipelines, its a tech preview feature

**Comment by jwforres** [comment]
@benjaminapetersen i'll let @adellape confirm but I don't think you want to touch images that were in the release notes for a previous release (release_notes directory)

**Comment by jwforres** [comment]
thanks for doing this! some of these pictures were *extremely* outdated

**Comment by benjaminapetersen** [comment]
@jwforres Aaahhh... yeah, would make sense to leave previous releases as previous images :)

No prob, this is tedium I can take off your plate!

**Comment by benjaminapetersen** [comment]
I'm back in office, just a refresh ping to get this cleaned up and finished off.

@adellape

**Comment by benjaminapetersen** [comment]
- removed the updated `release_notes` screenshots per @jwforres 
- replaced the `architecture/infrastructure_components/images/console_navigation.png` with a non-pipelines (overview) image.

@adellape

**Comment by adellape** [comment]
@benjaminapetersen Looks good, I'll try to come up with some text for those web_console.adoc callouts.

Can we chop off the banner in the cli_help.png image, since it specifically says "OpenShift Origin" there? That way it's generic enough to show up in all builds (OCP, Online, Dedicated, Origin).

File: architecture/infrastructure_components/web_console.adoc [hunk 1]
```diff
@@ -109,33 +109,18 @@ image::console_navigation.png["Web Console Project Overview"]
...
-pane displays detailed information about that object. In this example, the
-*cakephp-mysql-example* deployment is selected, and the *Details* pane is
-displaying details on the related
-xref:../core_concepts/deployments.adoc#replication-controllers[replication
-controller].
+<4> The *Applications* tab...
+<5> The *Builds* tab...
+<6> The *Resources* tab...
+<7> The *Storage* tab...
+<8> The *Monitoring* tab...
```
Comment (adellape) [inline architecture/infrastructure_components/web_console.adoc]: @benjaminapetersen @jwforres How bout something like:

```
<4> *Applications* tab: Browse and perform actions on your deployments, pods, services, and routes.
<5> *Builds* tab: Browse and perform actions on your builds and image streams.
<6> *Resources* tab: View your current quota consumption and other resources.
<7> *Storage* tab: View persistent volume claims and request storage for your applications.
<8> *Monitoring* tab: View logs for builds, pods, and deployments, as well as event notifications for all objects in your project.
```

Comment (benjaminapetersen) [inline architecture/infrastructure_components/web_console.adoc]: Works for me! updated.

**Comment by benjaminapetersen** [comment]
@adellape yup, chopped the banner on the `cli_help.png`.

**Pull request merged**